http-clients/src/test/java/org/triplea/http/client/web/socket/WebSocketConnectionTest.java

```diff
@@ -93,7 +128,16 @@ void queuedMessagesAreFlushedOnConnectionOpen() {
     @BeforeEach
     void setup() {
       webSocketConnection = new WebSocketConnection(INVALID_URI);
-      webSocketConnection.setClient(webSocketClient);
+    }
+
+    private void requiresSendTextAction() {
+      when(webSocket.sendText(any(), anyBoolean()))
+          .thenReturn(CompletableFuture.completedFuture(null));
+    }
+
+    private void requiresCloseAction() {
```

Comment:
This method seems out of order, it is invoked much further down in the test, right?

Second, why does this method make sense, what does requiring a close action mean and have to do with a send close returning a nullable future?

Reply:
The name is perhaps slightly misleading.
The idea is that tests that "require a close action"/run code that invokes sendClose at some point should call this method.

Suggestions for the name?

Reply:
Nevermind, I'll just inline this function, it is only used once anyways